Dockerfile

```diff
@@ -12,3 +12,8 @@ RUN ansible-galaxy collection install -r ${HOME}/requirements.yml \
 COPY watches.yaml ${HOME}/watches.yaml
 COPY roles/ ${HOME}/roles/
 COPY playbooks/ ${HOME}/playbooks/
+
+ENTRYPOINT ["/tini", "--", "/usr/local/bin/ansible-operator", "run", \
+    "--watches-file=./watches.yaml", \
+    "--reconcile-period=0s" \
+    ]
```

Comment:
Starting in operator-sdk v1.19.0, the default reconciliation interval was set to 5 minute, meaning the reconciliation loop would continuously run even though there were no changes.  The default in v1.22.x is 10 hours.  Our desired behavior is to only run the reconciliation loop when there have been changes to the CR being watched, or to it's dependent resources.  Therefore, we have set the reconcile-period to 0s, which means it will not be reconciled on an interval.  

Context: https://github.com/operator-framework/operator-sdk/issues/5655